e2e/verifiable-credentials.service.e2e.ts

```diff
@@ -54,8 +62,10 @@ describe('Verifiable credentials tests', () => {
     await signerService.publicKeyAndIdentityToken();
     rootOwnerDid = signerService.didHex;
 
+    const cacheClient = new CacheClient(signerService);
     verifiableCredentialsService = await getVerifiableCredentialsService(
-      signerService
+      signerService,
+      cacheClient
```

Comment:
Instead of full mock of a file you can just pass here an object with mocked methods.
```suggestion
      { getClaimsBySubject } as unknown as CacheClient
```

Reply:
This is simper of course, but I would prefer to mock according with Jest docs, because from this gives more flexibility and control as project grows